backend/src/main/java/me/performancereservation/domain/user/service/UserService.java

```diff
@@ -0,0 +1,47 @@
+package me.performancereservation.domain.user.service;
+
+import lombok.RequiredArgsConstructor;
+import me.performancereservation.domain.user.entitiy.User;
+import me.performancereservation.domain.user.enums.Role;
+import me.performancereservation.domain.user.repository.UserRepository;
+import me.performancereservation.global.exception.AppException;
+import me.performancereservation.global.exception.ErrorCode;
+import me.performancereservation.global.exception.ErrorType;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@Transactional
+@RequiredArgsConstructor
+public class UserService {
+
+    private final UserRepository userRepository;
+
+    //유저 정보 입력 (회원가입)
+    @Transactional
+    public User registerUser(String email, String name, String phoneNumber, Role role) {
```

Comment:
```suggestion
    public User register(String email, String name, String phoneNumber, Role role) {
```

이건 제안인데 UserService에서의 register이기 때문에 register()로도 의미가 명확한 것 같아요 
이건 선호하시는 방식으로 하셔도 될 것 같슴다

Reply:
말씀해주신 내용에 대해 고민해보았는데 Auth 에서도 registerAuth로 사용하고 있어 구분하는 방식으로 해보겠습니다!!